### ISSUE TITLE
Change RHCOS booting for AWS to work like GCP/Azure

### ISSUE THREAD

**Issue by cgwalters** [openshift/installer#2906]
For GCP and Azure and other clouds, all we upload is a blob to storage.  The installer creates "bootable images" from that.

But for historical reasons, the RHCOS pipeline creates AMIs directly - the installer just uses them for the bootstrap node, creating a separate [encrypted AMI](https://github.com/openshift/installer/pull/1296) for the actual cluster.

I think we should do the encryption bit before launching the bootstrap, and then RHCOS can stop publishing AMIs, and things will work more consistently across clouds.

**Comment by cgwalters** [comment]
Also xref this comment: https://github.com/coreos/mantle/pull/1168#discussion_r371181226

Basically...today the *COS pipelines only sanity check in AWS - for RHCOS it's convenient because we have AMIs, but as soon as we don't, this section will break:
https://docs.openshift.com/container-platform/4.3/installing/installing_aws/installing-aws-user-infra.html#installation-aws-user-infra-rhcos-ami_installing-aws-user-infra

Basically as the issue notes we want `openshift-install create-bootable` or so - until we have that, doing UPI installs on !AWS is a lot more painful and requires users to dive into the `rhcos.json`.

**Comment by cgwalters** [comment]
Although as of lately, https://github.com/openshift/installer/pull/3293 landed which pushes things back towards using AMIs.  So...I guess we should close this.
